docs/docs/migration-guide.mdx

```diff
@@ -55,6 +55,15 @@ We've updated our docs to reflect these changes. All examples now show a direct
 correct spaCy model. We've also added a warning to the [SpaCyNLP](components.mdx#spacynlp)
 docs that explains the fallback behavior.
 
+### Forms
```

Comment:
that needs to go in a different section now, doesn't it? 2.6 to 2.5

Reply:
Moved it! The description is ok?